_config.yml

```diff
@@ -1,3 +1,6 @@
+domain: okfnlabs.org
+url: http://okfnlabs.org
```

Comment:
Why use http instead of https?

Reply:
@augusto-herrmann the site still have lot of references to http. We need to move all of them to https before consider https done.
Now, https is available but you'll get some _mixed content_ errors.